app/models/user.rb

```diff
@@ -33,4 +33,16 @@ class User < ApplicationRecord
   def over_ta?
     %w[ta admin super].include? role
   end
+
+  def ensure_authentication_token
+   loop do
+      token = SecureRandom.urlsafe_base64(24).tr('lIO0', 'sxyz')
+      result = update(token: token)
+      break if result
+    end
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Lint/BlockAlignment: `end` at 42, 4 is not aligned with `loop do` at 38, 3.